steps/expval.py

```diff
@@ -86,22 +86,13 @@ def run_circuit_and_get_expval(
         instruction_qubits = [qubit.index for qubit in instr[1]]
         active_qubits.extend(instruction_qubits)
 
-
-    # Get the qubits we would like to measure
-    # Data for identities is not stored, need to account for empty terms
-    op_qubits = [term[0][0] for op in ops for term in op.terms if term]
-
-    need_to_activate = set(op_qubits) - set(active_qubits)
+    need_to_activate = set(range(len(qc.qubits))) - set(active_qubits)
     if need_to_activate:
+        # We activate all the qubits for convenience
```

Comment:
Makes sense, is there an underlying reason for this change? An update to how Orquestra behaves?

Reply:
Yes, some Orquestra packages were updated (e.g., `z-quantum-core`) and this workaround had to be adjusted to include an identity on all wires.